modules/gke-cluster-standard/variables.tf

```diff
@@ -113,6 +113,23 @@ variable "cluster_autoscaling" {
   }
 }
 
+variable "default_node_pool" {
+  description = "Enable default node-pool."
+  type = object({
+    remove_pool        = optional(bool, true)
+    initial_node_count = optional(number, null)
+  })
+  default = {
+    remove_pool        = true
+    initial_node_count = null
+  }
```

Comment:
Remove outer default, remove null default (it's implied), make non-nullable.
```hcl
  type = object({
    remove_pool        = optional(bool, true)
    initial_node_count = optional(number)
  })
  default = {}
  nullable = false
```

Reply:
Fixed in 835c2f9543ac93505a330bc836a4a590893883ee :pray: